src/components/Match/matchColumns.jsx

```diff
@@ -833,6 +833,13 @@ export default (strings) => {
       relativeBars: true,
       sumFn: true,
     },
+    {
+      displayName: "LH First 10 Mins",
+      tooltip: strings.lh_ten,
+      field: 'lh_t',
+      sparkline: true,
+      width: 240,
```

Comment:
I'm a bit weirded out that I *HAVE* to add the width *here*.  Otherwise any `recharts` chart that I add winds up putting in a big "pseudo-margin-right" that I can't get rid of.

Reply:
I took a look but couldn't find the cause. I'm fine with this as a workaround.